### DISCUSSION TITLE
Is this repository stale?

### DISCUSSION THREAD

**Discussion by MichaelShmalkoProCoder** [BlazorFluentUI/BlazorFluentUI#410] [General]
@limefrogyank Hi Lee, this project is probably the best UI components set for Blazor. But the repository is not mowing forward since late last year, is this code abandoned now?

Regards,
Michael

**Comment by limefrogyank** [comment]
It's functional, but we are not making new components. The FAST design components are how Microsoft is moving forward with fluent design web components.  While they're still a bit behind in component complexity, that's what MS is pushing for blazor (via another repo) and where most efforts are focused.

Sent from Surface Duo
________________________________
From: Michael Shmalko ***@***.***>
Sent: Monday, March 14, 2022 7:35:57 PM
To: BlazorFluentUI/BlazorFluentUI ***@***.***>
Cc: Lee McPherson ***@***.***>; Mention ***@***.***>
Subject: [BlazorFluentUI/BlazorFluentUI] Is this repository stale? (Discussion #410)


@limefrogyank<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Flimefrogyank&data=04%7C01%7C%7Ccfed69c2668b401dbddf08da062c8988%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637829085609505612%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=N9faY0ehOITEG%2BM8sxYjmgu8vgMpbiTRZplu9rn2WnA%3D&reserved=0> Hi Lee, this project is probably the best UI components set for Blazor. But the repository is not mowing forward since late last year, is this code abandoned now?

Regards,
Michael

**Reply by MichaelShmalkoProCoder** [reply]
Lee, thanks for the quick response! 

We are building forms and grids heavy application, that needs to be responsive and resemble a desktop experience as much as possible.
As someone who has a lot of visibility into the subject, would you be able to recommend a component library for a brand new Blazor WebAssembly application? 

Thank you!
Michael
